src/ui/components/CCItemUses.vue

```diff
@@ -44,7 +44,7 @@ export default class CCItemUses extends Vue {
     return this.item.getTotalUses(this.bonus)
   }
   get current(): number {
-    return this.max - this.item.MissingUses
+    return this.item.Uses
```

Comment:
Uses is set to the correct maximum already upon Full Repair; trust that it's accurate and use the getter directly.